examples/imagenet/ilsvrc12.h

```diff
@@ -0,0 +1,409 @@
+/************************************************************
```

Comment:
add a ilsrc12.cc to include a main function for create the bin files.
update cmake to add a new target for the main function.